docs/developer/pandas/partition_api.rst

```diff
@@ -0,0 +1,79 @@
+Pandas Partition API in Modin
+=============================
+
+If you are working with a Modin Dataframe, you can unwrap its remote partitions
+to get the raw future objects compatible with the execution engine (e.g. ``ray.ObjectRef`` for Ray).
+You can use this API to get the IPs of the nodes that hold these objects as well. In that case you can pass the partitions
```

Comment:
Create a separate section to talk about IPs. It is more of an advanced feature, so talk about it later in one place.

Reply:
Created the separate section for this.